core/services/fluxmonitor/flux_monitor.go

```diff
@@ -548,6 +615,45 @@ func (p *PollingDeviationChecker) consume() {
 	}
 }
 
+func (p *PollingDeviationChecker) performInitialPoll() {
+	if !p.initr.PollTimer.Disabled && !p.isHibernating {
+		p.pollIfEligible(DeviationThresholds{
+			Rel: float64(p.initr.Threshold),
+			Abs: float64(p.initr.AbsoluteThreshold),
+		})
+	}
+}
+
+func (p *PollingDeviationChecker) setPollTicker() {
+	if p.pollTicker != nil {
+		p.pollTicker.Stop()
+	}
+	if !p.initr.PollTimer.Disabled && !p.isHibernating {
+		p.pollTicker = time.NewTicker(p.initr.PollTimer.Period.Duration())
+		p.chPollTicker = p.pollTicker.C
+	}
+}
+
+// Hibernate restarts the PollingDeviationChecker in hibernation mode
+func (p *PollingDeviationChecker) Hibernate() {
+	logger.Infof("entering hibernation mode for contract: %s", p.initr.Address.Hex())
+	p.isHibernating = true
+	p.setPollTicker()
+	p.idleTimer = time.After(hibernationPollPeriod)
+	p.roundTimer = nil
+}
+
+// Reactivate restarts the PollingDeviationChecker without hibernation moce
+func (p *PollingDeviationChecker) Reactivate() {
+	logger.Infof("exiting hibernation mode, reactivating contract: %s", p.initr.Address.Hex())
+	p.isHibernating = false
+	p.pollIfEligible(DeviationThresholds{Rel: 0, Abs: 0})
+	if p.pollTicker != nil {
+		p.pollTicker.Stop()
```

Comment:
Maybe I'm lacking context, but doesn't we want to reactivate polling when `.Reactivate()` is called?